available-trials.md

```diff
@@ -8,6 +8,6 @@ Sign up to enable any of these features on your origin by filling out [this form
 | [Generic Sensors](https://developers.google.com/web/updates/2017/09/sensors-for-the-web) | Provides a framework that provides consistent interfaces for sensor devices exposed to the web platform. <br> [Intent to Experiment](https://groups.google.com/a/chromium.org/d/msg/blink-dev/2zPZt3watBk/M4qcI8wlBwAJ) | First available in Chrome M63. Trial ends on April 10, 2018. |
 | [Media Capabilities](https://wicg.github.io/media-capabilities/#decoding-encoding-capabilities) | Allow web sites to get more information about the decoding abilities of the device/system/browser, to enable more optimal decisions when selecting media streams for the user. <br> [Intent to Experiment](https://groups.google.com/a/chromium.org/forum/#!topic/blink-dev/dsdTLv33r4w) | First available in Chrome M64. Trial ends on April 12, 2018. |
 | [Disable Hardware Noise Suppression](https://developers.google.com/web/updates/2017/12/disabling-hardware-noise-suppression) | Allows WebRTC to rely on its internal noise suppressor, by disabling hardware noise suppression. Purpose is to test for an improvement in echo cancellation and audio quality. <br> [Intent to Experiment](https://groups.google.com/a/chromium.org/forum/#!topic/blink-dev/8QacQMv6oAY) | First available in Chrome M64. Trial ends on February 27, 2018. |
-| [AudioWorklet](https://webaudio.github.io/web-audio-api/#AudioWorklet) | The AudioWorklet exposes the low-level audio processing capability to web developers within Web Audio API's ecosystem. It started as a replacement of ScriptProcessorNode, which revealed serious design flaws over past years, but AudioWorklet offers low latency, runs on render thread and empowers developers with extensible interface. <br> [Intent to Experiment](https://groups.google.com/a/chromium.org/d/topic/blink-dev/oeBf3websgM) | First available in Chrome M64. Trial ends on March 6, 2018. |
+| [AudioWorklet](https://webaudio.github.io/web-audio-api/#AudioWorklet) | The AudioWorklet exposes the low-level audio processing capability to web developers within Web Audio API's ecosystem. It started as a replacement of ScriptProcessorNode, which revealed serious design flaws over past years, but AudioWorklet offers low latency, runs on render thread and empowers developers with extensible interface. <br> [Intent to Experiment](https://groups.google.com/a/chromium.org/d/topic/blink-dev/oeBf3websgM) | First available in Chrome 64. Trial ends on March 6, 2018. Shipping in Chrome 66, beta available March 15, 2018. |
```

Comment:
We should probably be consistent with the version naming within this file (Chrome 64 vs Chrome M64) -- otherwise, this looks good.